markdown/3.x/en/api/ff-filter-cloud.api.md

```diff
@@ -0,0 +1,13 @@
+## `ff-filter-cloud`
+___
+### Properties
+| Name | Description |
+| ---- | ----------- |
+| **blacklist** (String) (default: undefined) | A comma separated list of asn group names which should not appear in the list of selected filters. Changing this value causes ff-filter-cloud to update. It will not issue a request to FACT-Finder. This attribute cannot be used in conjunction with the whitelist attribute. Doing so will cause an error and the component will not render itself. |
```

Comment:
Let's always highlight names that appear in HTML.
```suggestion
| **blacklist** (String) (default: undefined) | A comma separated list of asn group names which should not appear in the list of selected filters. Changing this value causes `ff-filter-cloud` to update. It will not issue a request to FACT-Finder. This attribute cannot be used in conjunction with the `whitelist` attribute. Doing so will cause an error and the component will not render itself. |
```